deps.sh doesn't download GTest

**Describe the bug**
CMake cannot find GTest after executing deps.sh to get dependencies

**To Reproduce**
Download source zip file, execute deps.sh and run cmake

**Expected behavior**
deps.sh installs all dependencies, cmake generating Makefiles

**Screenshots**
Exact output of cmake:
```CMake Error at /home/stefan/source/ffmpeg-build3/workspace/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR
  GTEST_MAIN_LIBRARY)
Call Stack (most recent call first):
  /home/stefan/source/ffmpeg-build3/workspace/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /home/stefan/source/ffmpeg-build3/workspace/share/cmake-3.27/Modules/FindGTest.cmake:270 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  third_party/testing.cmake:57 (find_package)
  CMakeLists.txt:358 (include)


-- Configuring incomplete, errors occurred!

Failed to Execute cmake -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX=/home/stefan/source/ffmpeg-build3/workspace -DENABLE_SHARED=off -DCMAKE_BUILD_TYPE=Release -DJPEGXL_ENABLE_BENCHMARK=OFF -DJPEGXL_ENABLE_DOXYGEN=OFF -DJPEGXL_ENABLE_MANPAGES=OFF -DJPEGXL_ENABLE_JPEGLI_LIBJPEG=OFF -DJPEGXL_ENABLE_JPEGLI=OFF -DJPEGXL_TEST_TOOLS=OFF -DJPEGXL_ENABLE_JNI=OFF .
```

**Environment**
 - OS: Linux Mint / Ubuntu 22.04 LTS
 - Compiler version: gcc 11.4.0
 - CPU type: x86_64
 - cjxl/djxl version string: 0.10.2

**Additional context**
I would like to include libjxl in an automated script to build a static ffmpeg. Therefore git should be avoided, I have no problem compiling libjxl with git submodules, I want a solution without git.
I am not sure for what GTest is needed and if there is any flag to disable the need for it.

<!--
Currently github does not allow uploading files that end in `.jxl`, but when you
rename them for example as `image.jxl.jpg`, it will be possible to upload them
and also view them in browsers that are configured to support it.

See https://github.com/orgs/github-community/discussions/18139
-->

--- a/deps.sh
+++ b/deps.sh
@@ -15,6 +15,7 @@ MYDIR=$(dirname $(realpath "$0"))
 # update a git submodule.
 TESTDATA="873045a9c42ed60721756e26e2a6b32e17415205"
 THIRD_PARTY_BROTLI="36533a866ed1ca4b75cf049f4521e4ec5fe24727"
+THIRD_PARTY_GOOGLETEST="58d77fa8070e8cec2dc1ed015d66b454c8d78850"
 THIRD_PARTY_HIGHWAY="58b52a717469e62b2d9b8eaa2f5dddb44d4a4cbf"
 THIRD_PARTY_SKCMS="42030a771244ba67f86b1c1c76a6493f873c5f91"
 THIRD_PARTY_SJPEG="e5ab13008bb214deb66d5f3e17ca2f8dbff150bf"
@@ -82,6 +83,7 @@ EOF
   # Sources downloaded from a tarball.
   download_github testdata libjxl/testdata
   download_github third_party/brotli google/brotli
+  download_github third_party/googletest google/googletest
   download_github third_party/highway google/highway
   download_github third_party/sjpeg webmproject/sjpeg
   download_github third_party/skcms \

